docs/pytree_arguments.rst

```diff
@@ -0,0 +1,67 @@
+Pytree arguments to JAX functions
+=================================
+
+What is a pytree?
+^^^^^^^^^^^^^^^^^
+
+In JAX, a pytree is **a container of leaf elements and/or more pytrees**.
+Containers include lists, tuples, and dicts (JAX can be extended to consider
+other container types as pytrees, see the "JAX pytrees" notebook under "Advanced
```

Comment:
I cannot figure out how to get a :ref: working to link directly to this page, I'm open to any tips :)

Reply:
I figured it out!